x-pack/legacy/plugins/siem/server/kibana.index.ts

```diff
@@ -66,7 +66,14 @@ export const initServerWithKibana = (
       all: {
         api: ['siem'],
         savedObject: {
-          all: [noteSavedObjectType, pinnedEventSavedObjectType, timelineSavedObjectType],
+          all: [
+            'alert',
+            'action',
+            'action_task_params',
+            noteSavedObjectType,
+            pinnedEventSavedObjectType,
+            timelineSavedObjectType,
+          ],
           read: ['config'],
         },
         ui: ['show'],
```

Comment:
This will mean the alerting UIs will have to do something like: `const canShow = capabilities.get().siem.showAlerting || capabilities.get().monitoring.showAlerting`? There's a way we can automate this but want to confirm that we will have to go through the capabilities of each consumer?

Reply:
At the moment, yeah... Alerting is breaking from prior precedent where instead of being it's own "feature" it's conceptually a "sub-feature" of a bunch of other features. All [UI Capabilities](https://www.elastic.co/guide/en/kibana/current/development-plugin-feature-registration.html#_ui_capabilities) at the moment are "scoped" to the feature they're declared: 

> This object is namespaced by feature id. For example, if your feature id is “foo”, then your UI Capabilities are stored at uiCapabilities.foo. To access capabilities, import them from ui/capabilities

Conceptually, we'll want to change this model for Alerting so that features like SIEM are able to alter the "UI Capabilities" of the Alerting app itself.

Reply:
Thanks for the guidance, I will work with this.

Reply:
I created https://github.com/elastic/kibana/issues/52300 for us to "fix" this. It'll likely come up more frequently now that we have various applications becoming more integrated in other various applications.